content/en/examples/application/mysql/mysql-statefulset.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         - |
           set -ex
           # Generate mysql server-id from pod ordinal index.
-          [[ `hostname` =~ -([0-9]+)$ ]] || exit 1
+          [[ $HOSTNAME =~ -([0-9]+)$ ]] || exit 1
```

Comment:
Hi @Intimaria ,
From this change, I assume that this statement fails,
```
 [[ `hostname` =~ -([0-9]+)$ ]] || exit 1
```

Further along in the file, the same statement is used (line 53).

It is not clear that this change is required?

Reply:
Yes, this line fails, 
`[ `hostname` =~ -([0-9]+)$ ]] || exit 1]`

It succeeds in line 53 due to it being a different initContainer (not Oracle based).

Reply:
Hi @kbhawkey 
Yes, this statement fails in the mysql:5.7 initContainer, 
```
[ `hostname` =~ -([0-9]+)$ ]] || exit 1 
```

A similar statement succeeds in line 53, due to it being a different initContainer (not Oracle based, like mysql:5.7).